woocommerce/woocommerce-android
# Variable refund notice

Addresses #2000.

Based on the available refund options (fees, shipping and taxes), the refund notice is changed accordingly.

| Single option | Two options | Three options |
|-|-|-|
| ![image](https://user-images.githubusercontent.com/1522856/111843565-b8015600-8901-11eb-8872-be5735d8331e.png) | ![image](https://user-images.githubusercontent.com/1522856/111843591-c3548180-8901-11eb-8258-184189009c4e.png) | ![image](https://user-images.githubusercontent.com/1522856/111843620-cea7ad00-8901-11eb-9dd1-d1b18a93e2d9.png) |

**To test:**
1. Create orders with different combinations of items (fees, shipping and taxes)
2. Go to Order details and issue a refund
3. Verify the right combination of options is displayed in the notice (or the notice is hidden, if none are available)

Update release notes:
